.ci/run-extension-tests.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+# shellcheck disable=SC2016
+#
+# Copyright (c) 2012-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+set -e
+
+function installDeps() {
+    sudo pip install selenium
+    wget https://github.com/mozilla/geckodriver/releases/download/v0.29.0/geckodriver-v0.29.0-linux64.tar.gz
+    tar -xvzf geckodriver*
+    chmod +x geckodriver
+    sudo mv geckodriver /usr/local/bin/
+}
+
+function cloneExtension() {
+    EXTENSION_PROJECT_NAME=$(basename "$EXTENSION_REPO")
+    export EXTENSION_PROJECT_NAME
+
+    mkdir -p /tmp/projects/"$EXTENSION_PROJECT_NAME"
+    git clone "${EXTENSION_REPO}" /tmp/projects/"$EXTENSION_PROJECT_NAME"
+    cd /tmp/projects/"$EXTENSION_PROJECT_NAME"
+    git checkout tags/"${EXTENSION_REVISION}"
+    git status
+}
+
+function prepareDevfile() {
```

Comment:
let's move all the logic of this file to a github action

Reply:
@benoitf do you mean to create more steps in the action and each step will execute some part of this script?

Reply:
I was not clear enough

today you're using that script by doing:
```
name: Run VSCode extension's tests
        if: env.EXTENSION_REPO != ''
        run: |
          ./.ci/run-extension-tests.sh
```

but if you turn it to a github action, then we could just invoke it by calling

```
       if: env.EXTENSION_REPO != ''
        uses: che-incubator/vscode-extension-tests@next
```

(so plug-in registry is only a client of the action hosted elsewhere)

Reply:
and with github actions it's easy to get unit tests / code coverage / etc.